packages/satori-js/client.ts

```diff
@@ -270,4 +270,16 @@ export class Client {
       return Promise.resolve(response !== undefined);
     });
   }
+
+    /** Delete an Identity. */
+    async deleteIdentity(session: Session, id: string) {
```

Comment:
The session already has a user ID that you can pass to the inner delete function. I think identity id == user id?

Reply:
You are right. The backend impl was changed to operate as you say([here](https://github.com/heroiclabs/satori/commit/52b4a1558ebf9f35a05f97f94c20cea78208a384)).